src/app/modules/shared_models/member_model.nim

```diff
@@ -117,6 +119,15 @@ QtObject:
       result = newQVariant(item.pubKey)
     of ModelRole.DisplayName:
       result = newQVariant(item.displayName)
+    of ModelRole.PreferredDisplayName:
+      # ["localNickname", "ensName", "displayName", "alias"]
+      if item.localNickname != "":
+        return newQVariant(item.localNickname)
+      if item.ensName != "":
+        return newQVariant(item.ensName)
+      if item.displayName != "":
+        return newQVariant(item.displayName)
+      return newQVariant(item.alias)
```

Comment:
Actually one more thing: anywhere in this file, whenever you touch any of the 4 properties, you should also emit `dataUpdated(..., ModelRole.PreferredDisplayName.int)`